.github/workflows/build_slim_packages.yaml

```diff
@@ -43,16 +43,17 @@ on:
 
 jobs:
   linux:
-    runs-on: ${{ endsWith(github.repository, '/emqx') && 'ubuntu-22.04' || fromJSON('["self-hosted","ephemeral","linux","x64"]') }}
+    runs-on: ${{ endsWith(github.repository, '/emqx') && 'ubuntu-22.04' || fromJSON('["self-hosted","ephemeral","linux",${{ matrix.profile[3] }}]') }}
```

Comment:
We need more radical changes here.
In  current version, this job will always run on `ubuntu-22.04` if the repo is emqx/emqx.
May be just remove `endsWith(github.repository, '/emqx') && 'ubuntu-22.04'` and let it always use self-hosted runners. Not a big deal.

Reply:
True, it'll then work for our other in-sync forks as well.  But it might not work for other forks outside `emqx` org.

Perhaps we could change the condition to `github.repository_owner == 'emqx' && fromJSON(...) || 'ubuntu-22.04'`?